lua/damagelogs/server/rdm_manager.lua

```diff
@@ -592,8 +592,9 @@ net.Receive("DL_GetForgive", function(_, ply)
 		tbl.canceled = true
 		if tbl.status == RDM_MANAGER_WAITING then
 			tbl.status = RDM_MANAGER_FINISHED
-			tbl.conclusion = "(Auto) Canceled by the victim"
+			tbl.conclusion = Damagelog.ForcedLanguage == "" and "(Auto) Canceled by the victim" or TTTLogTranslate(_, "RDMManagerAuto").." "..TTTLogTranslate(_, "RDMCanceled")
```

Comment:
That's not working. When no language is forced it's always static English.
Why not use `tbl.conclusion = TTTLogTranslate(ply.DMGLogLang, "RDMManagerAuto").." "..TTTLogTranslate(ply.DMGLogLang, "RDMCanceled")`?

Reply:
Because that way the conclusion would be in the victim's, not the admins' (who are to work with the report), language. I thought that the reason why that line was not translated.

I assumed that if server had forced language then it would be safe to translate that line to that language, and if not, use english.

Reply:
You're right, leave it like this. Thanks!

Reply:
Actually I overlooked the fact that `TTTLogTranslate` function will use english if first argument is nil and server has no forced language, so checking here isn't necessary. I'll replace it with `TTTLogTranslate(_, "RDMManagerAuto").." "..TTTLogTranslate(_, "RDMCanceled")`